test-integration/test_integration/test_predict.py

```diff
@@ -97,3 +97,44 @@ def test_predict_with_remote_image(tmpdir_factory):
     # lots of docker pull logs are written to stdout before writing the actual output
     # TODO: clean up docker output so cog predict is always clean
     assert out.strip().endswith("hello world")
+
+
+def test_predict_in_subdirectory_with_imports(tmpdir_factory):
+    tmpdir = tmpdir_factory.mktemp("project")
+    subdir = tmpdir.mkdir("my-subdir")
+    with open(subdir / "predict.py", "w") as f:
```

Comment:
It'd be nice to just use fixtures on disk. Less fiddly to write, IDEs work, etc.

Reply:
We did "fixtures" on disk for `cog init`